x/programs/examples/lottery_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package examples
+
+import (
+	"context"
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	//go:embed testdata/target/wasm32-wasi/release/lottery.wasm
```

Comment:
this is fine for testing but testdata should just contain wasm blobs

```suggestion
//go:embed testdata/lottery.wasm
```